config/initializers/omniauth.rb

```diff
@@ -21,3 +26,11 @@
   new_path = "#{OmniAuth.config.path_prefix}/failure?error_type=#{message_key}&error_msg=#{error_description}"
   Rack::Response.new(['302 Moved'], 302, 'Location' => new_path).finish
 }
+
+if login_path.nil?
+    puts "No Auth Configured, App won't run"
+    raise StandardError, "Stopping Rails Application as no Auth has been configured"
+end
+
+AUTH_LOGIN_PATH=login_path
```

Comment:
I think we can probably remove this constant with some logic in `config/routes.rb`. I'm happy to see this changed in the short term though rather than right away.